mingw-w64-keystone/PKGBUILD

```
@@ -0,0 +1,60 @@
+# Maintainer: Ben Orchard <thefirstmuffinman@gmail.com>
+
+_realname=keystone
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=0.9.2
+pkgrel=1
+pkgdesc="Lightweight multi-platform, multi-architecture assembler framework"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
+depends=()
```

```suggestion
depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs")
```
Take a look at `runtime-dependencies` https://github.com/msys2/MINGW-packages/pull/11137/checks#step:9:1631

---

Thanks, I didn't catch that!